client/src/components/ProfileSettings/ProfileDetails/ProfileDetails.tsx

```diff
@@ -0,0 +1,90 @@
+                                <CardActionArea>
+                                    <CardMedia className={classes.cardImage} component="img" alt="profile details img" height="250" 
+                                    image={profCover} />
+                                    <Avatar className={classes.cardAvatar} alt="Remy Sharp" src={profimg} />
+                                </CardActionArea>
+                                <CardContent className={classes.contentWrapper}>
+                                    <Typography align="center" variant="h4">Norma Byers</Typography>
+                                    <Typography align="center" gutterBottom variant="body1">Loving Pet Sitter</Typography>
+                                    <Typography align="center" gutterBottom variant="body2">
+                                        <LocationOnIcon className={classes.locationIcon} color="secondary" fontSize="small"/>Toronto, Ontario
+                                    </Typography>
+                                    <Typography gutterBottom variant="h4">About me</Typography>
+                                    <Typography align="left" gutterBottom variant="subtitle1" component="h2">
+                                        Lorem ipsum dolor, sit amet consectetur adipisicing elit. Laborum nemo culpa dolore unde illum in quam rerum repellat. Sequi culpa atque,quam laboriosam minus possimus assumenda doloribus distinctio maxime nesciunt!
+                                    </Typography>
+                                    <Box display="flex" p={2} flexDirection="row" flexBasis="1">
+                                        <Avatar className={classes.thumbImages} variant="square" alt="Remy Sharp" src={doggo}/>
+                                        <Avatar className={classes.thumbImages} variant="square" alt="Remy Sharp" src={doggo1}/>
+                                    </Box>
+                                </CardContent>
+                            </Card>
+                        </Paper>
+                    </Grid>
+
+                    <Grid item>
+                        <Paper elevation={12} className={classes.availWrapper}>
+                            <Box p={5}>
+                                <Typography gutterBottom align="center" variant="h4">$14/hr</Typography>
+                                <Typography align="center">
+                                    <Rating name="simple-controlled" />
+                                </Typography>
+                                {/* HANDLING FORMDATA YET TO BE IMPLEMENTED */}
+                                <form>
+                                    <Box p={4}>
+                                        <Box>    
+                                            <Typography variant="caption" align="left">DROP IN</Typography>
+                                            <TextField id="datetime-local" type="datetime-local" className={classes.dateInput} />
```

Comment:
The id for the component is supposed to be a unique field, please change it here or in the component below.